testkitbackend/_sync/requests.py

```diff
@@ -119,13 +119,15 @@ def NewDriver(backend, data):
             backend, data["resolverRegistered"],
             data["domainNameResolverRegistered"]
         )
-    if data.get("connectionTimeoutMs"):
-        kwargs["connection_timeout"] = data["connectionTimeoutMs"] / 1000
-    if data.get("maxTxRetryTimeMs"):
-        kwargs["max_transaction_retry_time"] = data["maxTxRetryTimeMs"] / 1000
-    if data.get("connectionAcquisitionTimeoutMs"):
-        kwargs["connection_acquisition_timeout"] = \
-            data["connectionAcquisitionTimeoutMs"] / 1000
+    for timeout_testkit, timeout_driver in (
+        ("connectionTimeoutMs", "connection_timeout"),
```

Comment:
this would probably be easier to parse the other way around. it starts or R->L then assignment is done L<-R

Reply:
Do you have a strong opinion about this? While I see your point, I my mind this order makes sense as I mapping input from TestKit to corresponding driver config options. `in -> out`.

Reply:
It's testkit and so it's not a critical concern for me.